lib/Operation.php

```diff
@@ -177,6 +165,7 @@ public function onEvent(string $eventName, Event $event, IRuleMatcher $ruleMatch
 							'exception' => $e,
 						]
 					);
+					continue;
```

Comment:
continue at the end of try?
Doesn't look intended?

Reply:
At the end of catch, to skip in case of PlaceholderNotSubstituted exception, seems legit to me.

Reply:
github indentation threw me off...
The continue is inside the foreach